README.md

```diff
@@ -1,57 +1,72 @@
-# Getting Started with Wist
+# Wist
 
-Wist is a tool for identifying and reporting on patterns found in BrightScript code, with the goal of making code more consistent and avoiding bugs.
+<p align="center">
+  <img width= "200px" height = "200px" src="artwork/wist-logo.svg"/>
+</p>
 
-* Wist uses [ANTLR](https://www.antlr.org) for BrightScript parsing.
-* Wist uses an AST to evaluate patterns in code.
+Wist is a linter for identifying and reporting errors in Brightscript code. It helps you identify errors before you upload code to your Roku.
 
-## Installation and Usage
+> For more details. Please see our user guide and developer documentation [here](https://willowtreeapps.github.io/wist/)
```

Comment:
Can we make this **bold** instead of a blockquote? I think we want to bring attention to this section. Maybe also say "For full documentation, please see our user guide and developer guide here"?